src/language/kinds.js

```diff
@@ -47,3 +47,14 @@ export const DIRECTIVE = 'Directive';
 export const NAMED_TYPE = 'NamedType';
 export const LIST_TYPE = 'ListType';
 export const NON_NULL_TYPE = 'NonNullType';
+
+// Schema
```

Comment:
Similarly, should move to another file - clientKinds.js or something.

Or maybe this all goes into a subfolder? `language/client/*.js`